### PR TITLE
feat #CCOP-110: possibility to create review by authorized user

### PR DESCRIPTION
**Pull request by YanBrovkin**

### CODE REVIEW

File: assets/js/services.js [hunk 1]
```diff
@@ -108,6 +108,9 @@ storefrontApp.service('catalogService', ['$http', function($http, $localStorage)
         },
         searchCategories: function (criteria) {
             return $http.post('storefrontapi/categories/search', criteria);
+        },
+        addReview: function (productId, review) {
```
Comment (eMazeika) [inline assets/js/services.js]: redundant parameter "productId". It's in review already

File: assets/js/product.js [hunk 2]
```diff
@@ -49,6 +50,33 @@ storefrontApp.controller('productController', ['$rootScope', '$scope', '$window'
...
+                return;
+            }
+            if (!$scope.reviewDescription) {
+                return;
+            }
+            var review = {
+                productId: currentProduct.id,
+                content: $scope.reviewDescription,
+                rating: $scope.reviewRating,
+                createdDate: new Date()
```
Comment (eMazeika) [inline assets/js/product.js:64]: createdDate property doesn't exist

File: assets/js/product.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ storefrontApp.controller('productController', ['$rootScope', '$scope', '$window'
         $scope.addToWishlistDisabled = false;
         $scope.availableLists = null;
         $scope.listType = null;
+        $scope.averageRating = null;
```
Comment (eMazeika) [inline assets/js/product.js]: averageRating: property not used anywhere?

File: assets/js/product.js (hunk 2 above)
Comment (YanBrovkin) [reply assets/js/product.js:64]: Fixed. Now it's used for filling CreateDate property in request

File: assets/js/services.js (hunk 1 above)
Comment (YanBrovkin) [reply assets/js/services.js]: Fixed

File: assets/js/product.js (hunk 3 above)
Comment (YanBrovkin) [reply assets/js/product.js]: removed

**Pull request merged**